fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -32,6 +32,9 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
     The exponentiated gradient algorithm is described in detail by
     :footcite:t:`agarwal2018reductions`.
 
+    Read more in the \
+        `User Guide <https://fairlearn.org/main/auto_examples/plot_mitigation_pipeline.html>`_.
```

Comment:
we should never reference the website, it should be referencing something that sphinx can resolve, example:

https://github.com/scikit-learn/scikit-learn/issues/26927